script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -2541,7 +2546,7 @@ I never realized that the thing I had lost was nostalgia.
 [sha:1263d749e9ca9db1337c1e001b04a56e2ac4f64f]{
 -- Page 151, Offset 42420.
 -- 意識が覚めていく。
-My consciousness awakens.
+My gradually consciousness rouses.
```

Comment:
Word order

Reply:
my consciousness certainly wasn't roused at the time
probably stay clear of writing so late (I say as I'm writing this comment at 5 AM)